sphinx/util/osutil.py

```diff
@@ -72,20 +71,25 @@ def ensuredir(file: str | os.PathLike[str]) -> None:
     os.makedirs(file, exist_ok=True)
 
 
-def mtimes_of_files(dirnames: list[str], suffix: str) -> Iterator[float]:
-    for dirname in dirnames:
-        for root, _dirs, files in os.walk(dirname):
-            for sfile in files:
-                if sfile.endswith(suffix):
-                    with contextlib.suppress(OSError):
-                        yield path.getmtime(path.join(root, sfile))
+def _last_modified_time(source: str | os.PathLike[str], /) -> int:
+    """Return the last modified time of ``filename``.
 
+    The time is returned as integer microseconds.
+    The lowest common denominator of modern file-systems seems to be
+    microsecond-level precision.
 
-def copytimes(source: str | os.PathLike[str], dest: str | os.PathLike[str]) -> None:
```

Comment:
ditto

Reply:
Likewise: https://github.com/search?q=copytimes%28+NOT+is%3Aarchived+NOT+is%3Afork+language%3APython&type=code&p=1